libs/shared/db/mysql/account/src/lib/account.provider.ts

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Provider } from '@nestjs/common';
+
+import { MySQLConfig } from '../../../core/src';
+import {
+  AccountDatabase,
+  AccountDbProvider,
+  setupAccountDatabase,
+} from './setup';
+
+export const AccountDatabaseFactory: Provider<AccountDatabase> = {
```

Comment:
[non-blocking] can we name nest factories with the pattern `AccountDatabaseNestFactory`?